tests/integration/test_orchestration/calibrations/04_forth_node.py

```diff
@@ -0,0 +1,16 @@
+from qualibrate import QualibrationNode, NodeParameters
+
+
+class Parameters(NodeParameters):
+    qubits: list[str] = []
```

Comment:
Is this allowed? Doesn't it need to be a field?

Reply:
Allowed but this is bad practice honestly. This can cause issues related to closure.
I'll update.